src/main/resources/log4j2.properties

```diff
@@ -3,7 +3,7 @@ appender=console
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : %msg%n 
+appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : default} 
```

Comment:
it's missing %n at end of line in order to have logs in different lines. please take a look on it.

Reply:
Yes, sorry